### DISCUSSION TITLE
restor from backup.cbor

### DISCUSSION THREAD

**Discussion by PerrywLiu** [filecoin-project/lotus#7672] [Storage Provider] [answered]
when I restore from backup file backup.cbor, I use ./lotus-miner init restore /lotustmp/backup.cbor
it always sysnc show 

```
> State: idle; Current Epoch: 0; Todo: 0
> Validated 0 messages (0 per second)
```

and when I stop it , I saw the error :
`ERROR: restoring metadata: reading backup: unmarshaling log entry: cbor input should be of type array`

I do it in the same PC ,  does any step I fault? who can tell me ?

**Comment by SgtCoin** [comment]
@PerrywLiu I am not sure this is the issue, but I wanted to pass this along in case it help. 

From https://github.com/filecoin-project/lotus/issues/7302

>this can happen if you already have a backup, and then you run the backup again and overwrite the same file.
The recommended work-around is to delete the existing backup before making a new backup."

Please let me know if this works for you so I can document it for future reference.

**Comment by SgtCoin** [comment, marked as answer]
@PerrywLiu Please let us know if you are still working on this. I am going to archive it for now due to the age of the discussion.
